build/archives/BUILD.gn

```diff
@@ -107,6 +107,11 @@ if (build_engine_artifacts && flutter_prebuilt_dart_sdk) {
   zip_bundle("dart_sdk_archive") {
     deps = []
     output = "dart-sdk-$full_target_platform_name.zip"
+    if (is_mac) {
+      # Mac artifacts sometimes use mac and sometimes darwin. Standardizing the
```

Comment:
This comment is more of a TODO. Is there an issue to point at for it?

Reply:
Done: created https://github.com/flutter/flutter/issues/105351 to track.